src/data/filtering.js

```diff
@@ -27,6 +27,18 @@ export function isActive(filter) {
   return filter.active
 }
 
+
+/**
+ * Get a list of the active filter keys.
+ *
+ * @param {Object} filter
+ * @returns {string[]}
+ */
+export function getActiveFilterKeys(filter) {
+  const filterKeys = Object.keys(filter)
```

Comment:
no need for the intermediate `const` here, not sure it's adding anything.

Reply:
I suppose it's for the clarity that the additional identifier brings, but I'll inline it anyway.